[2023-03-29 23:40:25,474] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: weather_dag.extraction_weather_data manual__2023-03-30T02:40:20.974647+00:00 [queued]>
[2023-03-29 23:40:25,477] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: weather_dag.extraction_weather_data manual__2023-03-30T02:40:20.974647+00:00 [queued]>
[2023-03-29 23:40:25,477] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 23:40:25,477] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-29 23:40:25,477] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 23:40:25,482] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extraction_weather_data> on 2023-03-30 02:40:20.974647+00:00
[2023-03-29 23:40:25,487] {standard_task_runner.py:52} INFO - Started process 13787 to run task
[2023-03-29 23:40:25,490] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'extraction_weather_data', 'manual__2023-03-30T02:40:20.974647+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/var/folders/1t/ncp0s87s3bd37gfvks3bsb3w0000gn/T/tmpndms13pb', '--error-file', '/var/folders/1t/ncp0s87s3bd37gfvks3bsb3w0000gn/T/tmpnf4me7e8']
[2023-03-29 23:40:25,491] {standard_task_runner.py:80} INFO - Job 75: Subtask extraction_weather_data
[2023-03-29 23:40:25,517] {task_command.py:370} INFO - Running <TaskInstance: weather_dag.extraction_weather_data manual__2023-03-30T02:40:20.974647+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2023-03-29 23:40:25,541] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=extraction_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-03-30T02:40:20.974647+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-30T02:40:20.974647+00:00
[2023-03-29 23:40:26,426] {python.py:173} INFO - Done. Returned value was: None
[2023-03-29 23:40:26,434] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=extraction_weather_data, execution_date=20230330T024020, start_date=20230330T024025, end_date=20230330T024026
[2023-03-29 23:40:26,481] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 23:40:26,501] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
